pkg/template/apis/template/types.go

```diff
@@ -108,10 +108,24 @@ type TemplateInstanceSpec struct {
 // TemplateInstanceRequester holds the identity of an agent requesting a
 // template instantiation.
 type TemplateInstanceRequester struct {
-	// Username is the username of the agent requesting a template instantiation.
+	// username uniquely identifies this user among all active users.
```

Comment:
@jim-minter is right that it should be lower case (because it's supposed to match the json field name in this case).